src/super_gradients/training/transforms/transforms.py

```diff
@@ -538,8 +538,8 @@ def __call__(self, sample: dict):
             origin_img, origin_labels = sample["image"], sample["target"]
             cp_sample = sample["additional_samples"][0]
             img, cp_labels = cp_sample["image"], cp_sample["target"]
-
-            img, cp_labels = _mirror(img, cp_labels, 0.5)
+            cp_boxes = cp_labels[:, :4]
```

Comment:
Phew, nice job finding this bug!
Just a bit confused, `cp_boxes` isn't used anywhere below, you keep using `cp_labels`. Does it happen inplace or something?

Reply:
happens in place.

Reply:
Then why return something? Looks very bad

Reply:
Copying now the targets so that nothing happens in place, then plugging in labels.